src/templates/json.js

```diff
@@ -15,6 +15,20 @@ export default class JSONWrapper extends Component {
 		data: PropTypes.object,
 		location: PropTypes.object
 	};
+	constructor (props) {
+		super(props);
+		this.state = {
+			responseRenderModuleDescription: null,
+			responseRenderModuleMembers: null
+		};
+	}
+	async componentDidMount () {
+		const doc = JSON.parse(this.props.data.jsonDoc.internal.content);
+		this.setState({
+			responseRenderModuleDescription: await renderModuleDescription(doc),
+			responseRenderModuleMembers: await renderModuleMembers(doc[0].members)
+		});
+	}
```

Comment:
Note: `renderModuleDescription` and `renderModuleMembers` are now part of the state of component because they return a promise and we can't use `await` inside JSX. We get their values
in `ComponentDidMount` lifecycle and render their response in JSX.